loadtest/control/simplecontroller/controller.go

```diff
@@ -13,26 +13,28 @@ import (
 )
 
 type SimpleController struct {
+	id     int
 	user   user.User
 	stop   chan struct{}
 	status chan<- control.UserStatus
 	rate   float64
 }
 
-func (c *SimpleController) Init(user user.User) {
-	c.user = user
-	c.stop = make(chan struct{})
-	c.rate = 1.0
+func New(id int, user user.User, status chan<- control.UserStatus) *SimpleController {
```

Comment:
Let's add some godocs whenever we touch an exported function, so that it doesn't become a burden later :)

Reply:
Also created a dedicated [ticket](https://mattermost.atlassian.net/browse/MM-21389) so we try to improve on that front.